### ISSUE TITLE
Parsing error with SCSS : Colon is expected

### ISSUE THREAD

**Issue by skflowne** [UnwrittenFun/svelte-vscode#77]
I'm trying to make SCSS work with Sapper but the extension keeps giving me this error
```
Colon is expected svelte(css-syntax-error)
```
The code that triggers the error is pretty basic 
```
<style type="text/scss">
  .sass {
    h1 {
      color: red;
    }
  }
</style>
```
I did put this in `svelte.config.js` at the root of my project
```
const sass = require("node-sass")

module.exports = {
    preprocess: {
        style: async ({ content, attributes }) => {
            if (attributes.type !== "text/scss") {
                return
            }

            const result = await renderScss({
                data: content,
                sourceMap: "style.css.map",
                omitSourceMapUrl: true
            })

            return {
                code: result.css.toString("utf8"),
                map: result.map.toString("utf8")
            }
        }
    }
}
```
And added the node path in the extension settings

I'm using `svelte-preprocess` in `webpack.config.js` in case that matters

I'm not sure if `svelte.config.js` is just for the extension or if I need add its content into my `webpack.config.js`

Also this is a huge pain point in getting scss support to work, I'd like to help solve this so it works right away in the future

**Comment by skflowne** [comment]
Closing as there is already  #23 for this

**Comment by skflowne** [comment]
Reopened as I don't have a good way to make the extension work without having my svelte files with SCSS marked as containing errors.

I have unchecked all CSS related settings in hope to simply stop the extension from checking the CSS but it still does somehow.

I'm on WSL and I can't provide the path to the node executable that WSL uses, providing my windows node path does not make it stop from marking files either.

**Comment by kaisermann** [comment]
@skflowne can you provide a reproducible repo? that would help a lot

**Comment by skflowne** [comment]
So I've tried creating a whole new project (first one was on webpack) with rollup this time.

I've followed this [blog post](https://medium.com/@sean_27490/svelte-sapper-with-sass-271fff662da9) and here is the [repo](https://github.com/skflowne/sapper-scss-rollup)

Pretty sure the problem is I can't set the node path properly on WSL

I’ve tried setting it to the linux which path, setting it to my windows node, copying the linux node to a path that’s reachable by windows, but nothing works.

Any idea why it won't work with my windows node path ? It's not the same that WSL uses but it's node, I don't get why this makes a difference.
